File: cpp/examples/ops_registration.cpp
#include "registration_common.hpp"

int main() {
    auto set = examples::standard_fleet(
        {"atlas", "nova", "orion"},
        horus::core::RobotType::WHEELED,
        {0.8, 0.55, 0.45},
        "wheeled");
    return examples::register_or_fail(set);
}

File: rust/examples/ops_registration.rs
mod common;

use horus::{RobotDimensions, RobotType};

fn main() {
    let mut set = common::standard_fleet(
        &["atlas", "nova", "orion"],
        RobotType::Wheeled,
        RobotDimensions::new(0.8, 0.55, 0.45),
        "wheeled",
    );
    common::register_or_fail(&mut set);
}
